chaste_codegen/chaste_model.py

```diff
@@ -215,20 +212,20 @@ def _add_units(self):
             as well as units for converting membrane_stimulus_current."""
         for unit_name in self._UNIT_DEFINITIONS:
             try:
-                self._model.units.add_custom_unit(unit_name, self._UNIT_DEFINITIONS[unit_name])
-            except AssertionError:
+                self._model.units.add_unit(unit_name, self._UNIT_DEFINITIONS[unit_name])
```

Comment:
Do you want to have a separate PR that creates your own `UnitStore` or do it as part of this one?